dev/sg/linters/client.go

```diff
@@ -1,7 +1,39 @@
 package linters
 
-import "github.com/sourcegraph/sourcegraph/dev/sg/internal/lint"
+import (
+	"context"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/lint"
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/repo"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
 
 var (
 	inlineTemplates = lint.RunScript("Inline templates", "dev/check/template-inlines.sh")
 )
+
+func checkUnversionedDocsLinks() lint.Runner {
+	const header = "Literal unversioned docs links"
+
+	return func(ctx context.Context, s *repo.State) *lint.Report {
+		diff, err := s.GetDiff("**/*.tsx")
+		if err != nil {
+			return &lint.Report{Header: header, Err: err}
+		}
+
+		var mErr error
+		for file, hunks := range diff {
+			for _, hunk := range hunks {
+				for _, l := range hunk.AddedLines {
+					if strings.Contains(l, `to="https://docs.sourcegraph.com`) {
+						mErr = errors.Append(mErr, errors.Newf(`%s:%d: found link to 'https://docs.sourcegraph.com', use a '/help' link instead`,
```

Comment:
Ah! Took me a second to understand what the `/help` thing was, somehow it sounded like a special command, until https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+%22/help%22&patternType=literal showed me that it's just the path that we're talking about 😆 

```suggestion
						mErr = errors.Append(mErr, errors.Newf(`%s:%d: found link to 'https://docs.sourcegraph.com', use a '/help' relative path in the link instead`,
```